subsys/spm/spm.c

```diff
@@ -26,12 +26,12 @@
 #if USE_PARTITION_MANAGER
 #include <pm_config.h>
 #define NON_SECURE_APP_ADDRESS PM_APP_ADDRESS
+#define NON_SECURE_RAM_OFFSET PM_RAM_SECURE_SIZE
```

Comment:
Should the names `NON_SECURE_APP_ADDRESS` and `NON_SECURE_RAM_OFFSET` be given similar names ?

Reply:
This is in line with the original names. If you want to I can create a separate commit renaming the macros.

Reply:
thanks for explaining. accepted.